doc/source/release-notes/Percona-XtraDB-Cluster-5.6.29-25.15.rst

```diff
@@ -0,0 +1,78 @@
+.. rn:: 5.6.29-25.15
+
+===================================
+Percona XtraDB Cluster 5.6.29-25.15 
+===================================
+
+Percona is glad to announce the release of
+|Percona XtraDB Cluster| 5.6.29-25.19 on May 20, 2016.
+Binaries are available from the
+`downloads section <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.29-25.15/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.29-25.15 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.29-76.2 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.29-76.2.html>`_
+
+* `Galera Replicator 3.15 <https://github.com/codership/galera/issues?q=milestone%3A25.3.15>`_
+
+* `Codership wsrep API 25.15 <https://github.com/codership/mysql-wsrep/issues?q=milestone%3A5.6.29-25.15>`_
+
+All Percona software is open-source and free.
+
+For more information about relevant Codership releases, see `this announcement <http://galeracluster.com/2016/03/announcing-galera-cluster-5-5-48-and-5-6-29-with-galera-3-15/>`_.
+
```

Comment:
mention the LP milestone as well: https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.29-25.15